pom.xml

```
@@ -48,6 +48,14 @@
                     <linkXRef>false</linkXRef>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>17</source>
+                    <target>17</target>
```

```suggestion
                    <source>11</source>
                    <target>11</target>
```

Измени тут версию на 11, тогда должны пройти проверки Java CI